### DISCUSSION TITLE
What exactly is "val" in software output

### DISCUSSION THREAD

**Discussion by fredriklaa** [jakobrunge/tigramite#162] [Q&A]
First of all, congratulations on this fantastic implementation of the PCMCI/PCMCI+ algorithm!

I have one small question regarding the output from the software. You can find a column named "val". For ParCorr is seems to the effect size, while for RCOT it seems to be the test statistic. The reason I suspect this is that the "val" for ParCorr seems to be (relatively) constant for the same model but for different sample sizes, while for RCOT the "val" increases with increasing sample sizes.

In one of your papers it says that " the MCI test statistic also yields a well-interpretable notion of a normalized causal strength". This makes we a little more confused regarding what "val" is, since "val" then is supposed to be a normalized causal strength?

NB: Yes, RCOT is now *not* part of the standard installation of tigramite, but we managed to get it to work with the latest version of tigramite.

**Comment by jakobrunge** [comment]
It is always the test statistic value. For ParCorr it's the partial 
correlation, for GPDC the distance correlation after regressing out with a 
GP, for CMIknn the conditional mutual information. I'm not sure what you 
mean by effect size here. More precisely, it's the MCI-conditioned value 
which is further discussed in the Science advances paper.

Am 10. November 2021 02:37:40 schrieb fredriklaa ***@***.***>:

**Reply by fredriklaa** [reply]
Thanks for you prompt reply. With effect size I mean the causal "strength" of a relationship. I thought the partial regression coefficient told you the strength of a linear relationship between two variables, where one conditions "out" the remaining variables. Does this mean that the partial regression coefficient has a clear interpretation while the RCOT-statistic is a test statistic, and a test statistic in general does not need to have a "meaningful" interpretation? I have learnt that a test statistic always orders the sample space into "regions" with different "evidence" against the null hypothesis. 

Best
Fredrik

**Reply by jjakenichol** [reply]
I think you're confusing two similarly named quantities. A partial regression coefficient, as it seems you're familiar with, is a coefficient in the function found in a multiple linear regression. The partial correlation coefficient, which is what's used in TIGRAMITE, is documented well here: https://en.wikipedia.org/wiki/Partial_correlation. It also entails a regression, but only a simple linear regression. The correlation relates to how correlated two residuals are from the preceding regressions. This is the Pearson correlation coefficient on residuals - it's not a coefficient in a function.

**Reply by fredriklaa** [reply]
Hello,
Thanks for the link and the clarification! The article says that "The partial correlation coincides with the conditional correlation if the random variables are jointly distributed as the multivariate normal......" So in  general the two stated quantities are not the same, while in the gaussian case they are the same.
